### ISSUE TITLE
Color is lost 

### ISSUE THREAD

**Issue by IagoLast** [CartoDB/carto-vl#347]
## What is happening?
When I try to style the width by the `acress_lost` the `color` is not being applied.

## What should happen?
The color should be kept as when width is set to 10 and the size should be proportional to the acress lost column.

Example [here](https://cartodb.github.io/carto-vl/examples/editor/index.html#eyJhIjoiY3JvcF9sb3NzXzIwMTciLCJiIjoiIiwiYyI6ImNhcnRvZ2wiLCJkIjoiaHR0cHM6Ly97dXNlcn0uY2FydG8uY29tIiwiZSI6IndpZHRoOiAkYWNyZXNfbG9zdCAvIDEwMDBcbmNvbG9yOiByYW1wKGxpbmVhcigkYWNyZXNfbG9zdCwgMCwgMTAwMCksIFBSSVNNKSIsImYiOnsibG5nIjotOTkuMTc5ODQ3ODUwNTY4OTksImxhdCI6MzkuNzE3OTAzOTU0Njg4MDR9LCJnIjozLjU2MDU3MzcxMzM3NDU1MiwiaCI6IkRhcmtNYXR0ZXIifQ==)
[![https://gyazo.com/2e6412a5d128746432375dce2b0699e1](https://i.gyazo.com/2e6412a5d128746432375dce2b0699e1.gif)](https://gyazo.com/2e6412a5d128746432375dce2b0699e1)

**Comment by davidmanzanares** [comment]
I don't think this is a bug. You're seeing big purple points since you are setting the style to: "make the point width proportional to the acres lost and make the point color purple when the acres lost is more than 1000"
